indra/tests/test_statements_serialization.py

```diff
@@ -1,88 +1,99 @@
 from __future__ import absolute_import, print_function, unicode_literals
-from builtins import dict, str
```

Comment:
I can't remember if I asked why you weren't using the `six` module to take care of these things automatically. Was there a problem with unicode/bytes?